docs/code_of_conduct.md

```diff
@@ -20,13 +20,22 @@ Private or public harassment of any kind will not be tolerated. Since harassment
 
 Additionally, spam and other content which disrupts or prevents LDK contributors from working on LDK is not acceptable.
 
+In case of a concrete issue that raises a deadlock about the application of the code of conduct, 2 additional members should be selected by consensus of the code of conduct team from the community of
+active LDK contributors to join the team in its decision-making process to adjudicate the case. The decision of the majority of the 5 members is binding.
+
 ## The Code of Conduct Team
 
 A small team of LDK contributors has volunteered to enforce the LDK Code of Conduct. If you feel like a community member has engaged in inappropriate behavior, please don't hesitate to contact one of the following LDK contributors via email or on Discord:
  * Matt Corallo - ldkcocpoc on mattcorallo.com
  * Val Wallace - vwallace on protonmail.com
  * Devrandom - devrandom99 on proton.me
 
+The Code of Conduct team should be renew every 2 years with a transparent nomination process opened to the whole community of active LDK contributors.
```

Comment:
> One of the biggest ways in euch codes of Conduct have gone wrong is when they're selected out of individuals who are super interested in being on the code of conduct committee

I fully agree and I think committee members should be kept selected among technical contributors to maintain the alignment with the community, rather than their own goal.

Proposing the following rule to be part of the committee:
"Code of conduct team members should have realized at least 50 code commits in the main LDK git tree and the first commit should have been merged more than 2 years at the date of nomination.”

50 code commits is around the current bar than the number of commits by the committee member with the lowest number of commits (Devrandom - 67 if i’m using git correctly) and the 2 years rule request than someone has been around the project for long enough to be familiar with the social norms of the communication space.

The requirement of a technical proof-of-work also ensure it stays a project maintained by developers for developers, and not a bureaucracy talking about stupid trademarks issues like what is happening with Rust. 

There is another pitfall that I think we should avoid it’s when people are no more technically active in the project, yet still occupying a janitorial role like a CoC committee member. A time-bounded nomination process allow those people to leave their roles smoothly and in a transparent fashion for the community. Nothing worst than “elected-for-life” roles.

I like the idea than to exercise a role you must be technically active, it’s okay to not do review and land code for few months though if after one or maybe two years you’re no more active, it’s better for everyone to transmit roles. It’s kinda FOSS spirit imho.

Removed the mention of a formal selection process (“Nomination process should be defined in its ulterior document”), I think proposing people in an open fashion on usual communication channels like we’re doing on Core-side for admin roles, sounds good enough to me.